src/main/java/com/commercetools/web/bind/annotation/PostJsonRequestJsonMapping.java

```diff
@@ -0,0 +1,37 @@
+package com.commercetools.web.bind.annotation;
+
+import org.springframework.core.annotation.AliasFor;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+/**
+ * Custom annotation for {@code POST} endpoints which accept (consume) and response (produce)
+ * <i>application/json</i> mime-type.
+ *
+ * @see PostRequestJsonMapping
+ */
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+@PostRequestJsonMapping(consumes = APPLICATION_JSON_VALUE)
```

Comment:
What do you think of name `PostJsonRequestJsonResponse`?

Reply:
like this 3c2656e ?

Reply:
What does `Mapping` means?

Reply:
> Annotation for mapping web requests onto specific handler classes and/or handler methods. Provides a consistent style between Servlet and Portlet environments, with the semantics adapting to the concrete environment.

Reply:
I find this `Mapping` in the name a bit redundant. It's enough to know that you use POST and the request should be JSON and response should be also JSON. But if you think it has meaning, you can leave it as it is.